packages/destination-actions/src/destinations/salesforce/sf-object-to-shape.ts

```diff
@@ -0,0 +1,27 @@
+import { GenericPayload } from './sf-types'
+
+export const mapObjectToShape = (payload: GenericPayload, sobject: string) => {
+  const shapeFunction = object_to_shape.get(sobject.toLowerCase())
```

Comment:
```suggestion
  const shapeFunction = objectToShape.get(sobject.toLowerCase())
```

Segment style is typically camelCase